telos-frontend/src/views/journal/Journal.css

```diff
@@ -1,17 +1,46 @@
 .Journal {
-    background-image: url("./journal.png");
-    background-size: 100% 100%;
-    width: 100%;
-    height: 100%;
+  background-image: url('./journal.png');
+  background-size: 100% 100%;
+  width: 100%;
+  height: 100%;
+  flex-direction: row;
+  display: flex;
+}
+
+.Half-journal {
+  display: flex;
+  flex-direction: column;
+  width: 50%;
+  margin-top: 10px;
+  margin-bottom: 15px;
+}
+
+.Half-journal:first-child {
+  margin-left: 25px;
+}
+
+.Half-journal:last-child {
+  margin-right: 18px;
 }
 
 .Arrow-left {
-    position: fixed;
-    bottom: 25px;
-    left: 28px;
+  position: absolute;
```

Comment:
This absolute positioning shouldn't be a problem as long as the page doesn't scroll once the top bar is rendered above it

Reply:
^^ Something we'll need to ensure